config/packages/doctrine.yaml

```diff
@@ -1,29 +1,42 @@
-parameters:
-    # Adds a fallback DATABASE_URL if the env var is not set.
-    # This allows you to run cache:warmup even if your
-    # environment variables are not available yet.
-    # You should not need to change this value.
-    env(DATABASE_URL): ''
-
 doctrine:
     dbal:
         url: '%env(resolve:DATABASE_URL)%'
 
-        # configure these for your database server
-        driver: 'pdo_mysql'
-        server_version: '10.6.7-MariaDB'
-        charset: utf8mb4
-        default_table_options:
-            collate: utf8mb4_unicode_ci
-
+        # IMPORTANT: You MUST configure your server version,
+        # either here or in the DATABASE_URL env var (see .env file)
+        server_version: 'mariadb-10.6.7'
```

Comment:
We should define this in connecting string to enable chances in helm charts without need to make new releases, if updating the database server.